applications/chats/consumers.py

```diff
@@ -0,0 +1,62 @@
+from channels.db import database_sync_to_async
+from channels.exceptions import DenyConnection
+from channels.generic.websocket import AsyncJsonWebsocketConsumer
+
+
+@database_sync_to_async
+def get_room(user, room_id):
+    return user.room_set.filter(id=room_id).first()
```

Comment:
Is a room_id unique?
I think it should be. In that case you can simply execute `get` instead of `filter`